### PR TITLE
feat(test): Add basic test cases

### PR DESCRIPTION
**Pull request by gaocegege**
Signed-off-by: Ce Gao <gaoce@caicloud.io>

**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #6 

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by gaocegege** [comment]
Please review https://github.com/caicloud/ormb/pull/61/commits/d07850570e52de4b81cb7735bee6c29804e424b6

File: pkg/saver/saver.go [hunk 1]
```diff
@@ -16,24 +16,20 @@ import (
...
-type defaultSaver struct {
+// Saver is the implementation.
+type Saver struct {
 	Parser parser.Parser
 }
 
-// NewDefaultSaver creates a new defaultSaver.
-func NewDefaultSaver() Saver {
-	return &defaultSaver{
+// New creates a new defaultSaver.
```
Comment (simon-cj) [inline pkg/saver/saver.go:24]: comment is not right now

File: pkg/ormb/ormb.go [hunk 2]
```diff
@@ -8,9 +8,9 @@ import (
 	"github.com/caicloud/ormb/pkg/saver"
 )
 
-// ORMB is the interface to save/pull/push/export
+// Interface is the interface to save/pull/push/export
```
Comment (simon-cj) [inline pkg/ormb/ormb.go:11]: `save/pull/push/export` -> should `login/save/pull/push/export/remove`

**Comment by gaocegege** [comment]
Comments are addressed. PTAL

**Pull request merged**